modules/howtos/pages/n1ql-queries-with-sdk.adoc

```diff
@@ -3,7 +3,7 @@
 :page-topic-type: howto
 :page-aliases: n1ql-query,ROOT:querying-n1ql
 :lang: Java
-:version: 3.1.0
+:version: 3.1.1
```

Comment:
Technical debt - we should be using the attributes.adoc file in project-docs to pull this in.
If we even need it in the doc?

Reply:
I thought you added this field :D  It drives me nuts changing the version numbers everywhere, if there's a better way to do it then yes please!

Reply:
Track fix at DOC-7969